api/message.go

```diff
@@ -371,6 +371,9 @@ func (a *MessageAPI) CreateMessage(ctx *gin.Context) {
 		if strings.TrimSpace(message.Title) == "" {
 			message.Title = application.Name
 		}
+		if message.Priority <= 0 {
```

Comment:
This would mean, that it's impossible to set a priority of 0 when a default priority is set. I'm not really sure how to proceed with this, because every client will probably send a priority, so then the default priority would never be used if we'd only use the priority if it's truly unset.

Reply:
That makes sense. I could update this to check the actual request body prior to binding it to a struct. That would allow a client to send a priority of 0.

This is really a benefit for those clients that don't support a priority or modification of the request body directly. Grafana alerts, for example, fit this use case.